LeetCodeSolutions/Program.cs

```diff
@@ -1,3 +1,22 @@
-﻿// See https://aka.ms/new-console-template for more information
-
-Console.WriteLine("Hello, World!");
\ No newline at end of file
+﻿namespace LeetCodeSolutions;
+public class Program
+{
+    public int Maximum69Number (int num)
+    {
+        if (num == 9999) return num;
```

Comment:
If this if statement was removed, would the code return the wrong result when the input is 9999?

Reply:
No, it would return the correct result. But I figured It would be better to return early if there wasn't any change needed on the given number.